packages/ui-components/.babelrc.js

```diff
@@ -26,5 +26,11 @@ module.exports = {
         loose: true,
       },
     ],
+    [
+      '@babel/plugin-proposal-private-property-in-object',
```

Comment:
Fixes annoying console output:
```
[ui-components] Though the "loose" option was set to "false" in your @babel/preset-env config, it will not be used for @babel/plugin-proposal-private-property-in-object since the "loose" mode option was set to "true" for @babel/plugin-proposal-class-properties.
[ui-components] The "loose" option must be the same for @babel/plugin-proposal-class-properties, @babel/plugin-proposal-private-methods and @babel/plugin-proposal-private-property-in-object (when they are enabled): you can silence this warning by explicitly adding
[ui-components] 	["@babel/plugin-proposal-private-property-in-object", { "loose": true }]
[ui-components] to the "plugins" section of your Babel config.
```